velox/exec/tests/SortBufferTest.cpp

```diff
@@ -611,13 +618,63 @@ DEBUG_ONLY_TEST_F(SortBufferTest, reserveMemoryGetOutput) {
     // Sets an extreme large value to get output once to avoid test flakiness.
     sortBuffer->getOutput(1'000'000);
     if (spillEnabled) {
-      ASSERT_EQ(numReserves, 1);
+      // Reserve memory for sort and getOutput.
+      ASSERT_EQ(numReserves, 2);
     } else {
       ASSERT_EQ(numReserves, 0);
     }
   }
 }
 
+DEBUG_ONLY_TEST_F(SortBufferTest, reserveMemorySort) {
+  struct SortTestOption {
```

Comment:
```
  struct {
    bool usePrefixSort;
    bool spillEnabled;
    
    std::string debugString() {...
  } testSettings[] = {
     {}
  }
```

Reply:
We cannot use `for (const auto [usePrefixSort, spillEnabled] : testSettings)` if we change to debugString, so I change it to current implement, ` SCOPED_TRACE(fmt::format(
        "usePrefixSort: {}, spillEnabled: {}, ", usePrefixSort, spillEnabled))`. Thanks!